src/Metrics/CharacterErrorRate.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace ItkDev\SentenceSimilarityMetrics;
+
+class CharacterErrorRate
+{
+    /**
+     * Calculates character error rate (CER) between two sentences.
+     */
+    public function cer(string $reference, string $hypothesis): float
+    {
+        $referenceCharacters = mb_str_split($reference); // Split into characters
```

Comment:
Normally I would like comments on the line before the code it comments on?

Reply:
Updated comments!